sdk/tables/azure-data-tables/README.md

```diff
@@ -222,13 +235,26 @@ the client level to enable it for all requests.
 
 
 ## Troubleshooting
+
 ### General
-Azure Data Tables clients raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md).
-All Table service operations will throw a `HttpResponseError` on failure with helpful [error codes](https://docs.microsoft.com/rest/api/storageservices/table-service-error-codes).
+Azure Data Tables clients raise exceptions defined in [Azure Core][azure_core_readme].
+When you interact with the Azure table library using the Python SDK, errors returned by the service respond ot the same HTTP status codes for [REST API][tables_rest] requests. The Table service operations will throw a `HttpResponseError` on failure with helpful [error codes][tables_error_codes].
+
+For examples, if you try to create a table that already exists, a `409` error is returned indicating "Conflict".
+```python
+service_client = TableServiceClient(connection_string)
 
+# Create the table if it does not already exist
+tc = service_client.create_table_if_not_exists(table_name)
+
+try:
+    service_client.create_table(table_name)
+except HttpResponseError:
+    print("Table with name {} already exists".format(table_name))
+```
 ### Logging
 This library uses the standard
-[logging](https://docs.python.org/3/library/logging.html) library for logging.
+[logging][python_logging] library for logging.
```

Comment:
not actually sure this logging code works, since I think the logger should actually be created for 'azure' (i.e. logger = logging.getLogger('azure')), can you double check? It'd also be nice to include a client call after creating your client, that way the logging will actually occur